CMakeLists.txt

```diff
@@ -176,6 +176,10 @@ if(WIN32)
     option(LLGL_BUILD_RENDERER_DIRECT3D12 "Include Direct3D12 renderer project" ON)
 endif()
 
+if(LLGL_TARGET_PLATFORM STREQUAL "Linux")
```

Comment:
`LLGL_TARGET_PLATFORM` in the CMake script is only intended as output for the summary, not for conditionals. You can  use the CMake variable `UNIX` for this condition instead.